changelogs/fragments/tmp-hlq-zos_data_set.yml

```diff
@@ -0,0 +1,7 @@
+minor_changes:
+   - >
+     zos_mvs_raw - Ensures that temporary datasets created by zos_data_set use the tmp_hlq specified.
+     This allows for a user to specify the data set high level qualifier (HLQ)
+     used in any temporary data set created by the module. Often, the
+     defaults are not permitted on systems, this provies a way to override
```

Comment:
Just a minor typo here where it says `provies` instead of `provides`

Reply:
Nice catch

Reply:
Thanks @rexemin - for double checking, when I saw the interface changes that was the first thing that came to mind, thanks for reviewing as well!!!